snaps/features/custom-ui/index.md

```diff
@@ -66,12 +66,16 @@ The following custom UI components are available:
 Outputs a formatted text field for a blockchain address. 
 The address is automatically displayed with a [Jazzicon](https://www.npmjs.com/package/@metamask/jazzicon)
 and truncated value. 
-Hovering over the address shows the full value in a tooltip.
+When the address is truncated, hovering over the address shows the full value in a tooltip.
```

Comment:
```suggestion
```
This isn't true

Reply:
Oops! So hovering always shows the full value?

Reply:
We don't have the tooltip anymore at all 😅

We are considering other ways to provide the full address when truncated